digdag-standards/src/main/java/io/digdag/standards/command/EcsCommandExecutor.java

```diff
@@ -618,6 +648,7 @@ protected void setEcsContainerOverrideCommand(
         }
         bashArguments.add(s("tar -zcf %s  --exclude %s --exclude %s .digdag/tmp/", outputProjectArchivePathName, relativeProjectArchivePath.toString(), outputProjectArchivePathName));
         bashArguments.add(s("curl -s -X PUT -T %s -L \"%s\"", outputProjectArchivePathName, outputProjectArchiveDirectUploadUrl));
+        bashArguments.add(s("echo %s", ECS_TASK_FINISHED));
```

Comment:
This needs escaping:
```suggestion
        bashArguments.add(s("echo \"%s\"", ECS_TASK_FINISHED));
```

Reply:
I added escape here.
https://github.com/treasure-data/digdag/pull/1316/commits/ec0e18ecc8a2133b5efbd17e26fddb6ccb21bb04